### PR TITLE
chore(CSI): Run CSharpier on all CSI related projects

### LINKED ISSUE #2942: Enable CSharpier on ETABS project

### PR DESCRIPTION
**Pull request by AlanRynne**
Fixes #2942

Changes resulting in performing a csharpier run on the `ConnectorCSI` and `Objects/Converters/ConverterCSI/`

Reverted changes made to Core.csproj as it was leading down a CI rabbit hole that had nothing to do with this issue.

### CODE REVIEW

**Review by JR-Morgan — APPROVED** [review]

File: Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs [hunk 1]
```diff
@@ -19,9 +19,10 @@ public partial class ConverterCSI
     readonly string Send2DResults = "send2DResults";
 
     private string _modelUnits;
+
     public string ModelUnits()
     {
-      if (_modelUnits != null) 
+      if (_modelUnits != null)
         return _modelUnits;
```
Comment (BovineOx) [inline Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs:26]: I'd even go with brackets here tbh, for single line - do we at least have line space after?

Comment (AlanRynne) [reply Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs:26]: CSharpier doesn't remove brackets or any other potentially dangerous changes, just enforces consistent spacing, line-breaks, etc...  

We can also enforce higher level style's such as disallowing single line if statements without brackets with the current setup, but we'd use dotnet format or resharper for that (resharper is waaay better, but not fast on CLI)

Devs already get info messages on their code based on the current preference (which i think is no brackets for single-line statements), but this is not really enforced.

File: Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs [hunk 2]
```diff
@@ -47,7 +50,10 @@ public static List<string> GetAllFrameNames(cSapModel model)
         model.FrameObj.GetNameList(ref num, ref names);
         return names.ToList();
       }
-      catch { return null; }
+      catch
```
[the remark this answers is not in the document]
Comment (AlanRynne) [reply Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs:53]: Yep.... these null returns are a PITA. But for now I'm happy with consistent line breaks on catch statements 😄

**Review by BovineOx — APPROVED** [review]
Have checked to a reasonable level and appears to be all style changes as discussed :)

**Pull request merged**